joschahenningsen/TUM-Live
# Fixed PWA Branding

### Motivation and Context


Currently, the PWA Logo is not correctly set. It is the default image, which is probably not what is wanted.
While not being a major issue, it is just one minor annoyance less 😉 
Resolves #711 

### Description

I added the icons to the branding directory
I added the relevant icons to the router

(On the way, I also took the liberty to make the default favicon.ico 48x48 instead of 16x16)

### Steps for Testing

Prerequisites: Probably none

1. Change the branding
   - [ ] [branding/icons-192.png](https://user-images.githubusercontent.com/26258709/208136764-19904aea-530c-4d17-9615-151313cfaa06.png)
   - [ ] [branding/icons-512.png](https://user-images.githubusercontent.com/26258709/208136771-dcfc7a53-28a4-474e-b9aa-02a005e670ff.png)
2. Install App as PWA
3. See updated Logo
4. ...
5. Profit

### Screenshots
